src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return first => {
+    return second => {
+      count++;
+
+      if (count >= 3 && count % 2 === 0) {
```

Comment:
should be precisely 3 successful calls
```suggestion
      if (count > 3 && count % 2 === 0) {
```